### PR TITLE
Add description about status and types

### PR DESCRIPTION
**Pull request by kjunu**
Please review English.

### CODE REVIEW

File: AIPS/aip-1.md [hunk 1]
```diff
@@ -10,3 +10,26 @@ created: 2019-08-08
 # What is an AIP?
 
 AIP stands for Aergo Improvement Proposal. An AIP is a design document providing information to the Aergo community, or describing a new feature for Aergo or its processes or environment. The AIP should provide a concise technical specification of the feature and a rationale for the feature. The AIP author is responsible for building consensus within the community and documenting dissenting opinions.
+
+# AIP Status ( Work Flow )
```
Comment (graup) [inline AIPS/aip-1.md]:
```suggestion
# AIP Status (Work Flow)
```

Comment (graup) [inline AIPS/aip-1.md]:
```suggestion
# AIP Status (Work Flow)
```

Comment (graup) [inline AIPS/aip-1.md]:
```suggestion
# AIP Status (Work Flow)
```

File: AIPS/aip-1.md [hunk 2]
```diff
@@ -10,3 +10,26 @@ created: 2019-08-08
...
+
+Basically anyone can suggest their ideas to Aergo community.
```
Comment (graup) [inline AIPS/aip-1.md]:
```suggestion
Everyone can suggest their ideas to the Aergo community.
```

File: AIPS/aip-1.md [hunk 3]
```diff
@@ -10,3 +10,26 @@ created: 2019-08-08
...
+
+ 1. None
```
Comment (graup) [inline AIPS/aip-1.md]:
```suggestion
 1. Start
```

File: AIPS/aip-1.md [hunk 4]
```diff
@@ -10,3 +10,26 @@ created: 2019-08-08
...
+     * Write improvement idea and create a pull request on Github (https://github.com/aergoio/aips).
+ 2. Active
+     * Aergo team will review the pull request and discuss on the GitHub.
```
Comment (graup) [inline AIPS/aip-1.md]:
```suggestion
     * Aergo team will review the pull request and discuss on GitHub.
```

File: AIPS/aip-1.md [hunk 5]
```diff
@@ -10,3 +10,26 @@ created: 2019-08-08
...
+ 3. Develop
+     * If idea is accepted. Aergo Team start to develop.
```
Comment (graup) [inline AIPS/aip-1.md]:
```suggestion
     * If idea is accepted, Aergo Team starts to develop.
```

File: AIPS/aip-1.md [hunk 6]
```diff
@@ -10,3 +10,26 @@ created: 2019-08-08
...
+ 4. Applied
+     * It will be applied on the mainnet by code.
```
Comment (graup) [inline AIPS/aip-1.md]:
```suggestion
     * After the code is released and tested, the mainnet will be upgraded.
```

File: AIPS/aip-1.md [hunk 7]
```diff
@@ -10,3 +10,26 @@ created: 2019-08-08
...
+
+# AIP Type
+
+There are n types of AIP:
```
Comment (graup) [inline AIPS/aip-1.md]:
```suggestion
There are 3 types of AIP:
```

File: AIPS/aip-1.md [hunk 8]
```diff
@@ -10,3 +10,26 @@ created: 2019-08-08
...
+ 1. Standard
+     * Describes any change that affects most or all Aergo implementations.
+ 2. Parameter
+     * Describes a blockchain parameters. for example, changing the number of block producers.
```
Comment (graup) [inline AIPS/aip-1.md]:
```suggestion
     * Describes a blockchain parameter. for example, changing the number of block producers.
```

**Review by graup — CHANGES_REQUESTED** [review]

**Comment by kjunu** [comment]
@graup Thank you :)

File: AIPS/aip-1.md [hunk 9]
```diff
@@ -10,3 +10,26 @@ created: 2019-08-08
...
+ 4. Applied
+     * After the code is released and tested, the mainnet will be upgraded.
+
+# AIP Type
+
+There are 3 types of AIP:
+ 1. Standard
+     * Describes any change that affects most or all Aergo implementations.
+ 2. Parameter
+     * Describes a blockchain parameter. for example, changing the number of block producers.
```
Comment (graup) [inline AIPS/aip-1.md:33]:
```suggestion
     * Describes the change of a blockchain parameter. For example, changing the number of block producers.
```

File: AIPS/aip-1.md [hunk 10]
```diff
@@ -10,3 +10,26 @@ created: 2019-08-08
...
+# AIP Status (Work Flow)
+
+Everyone can suggest their ideas to the Aergo community.
+
+ 1. Start
+     * Write improvement idea and create a pull request on Github (https://github.com/aergoio/aips).
+ 2. Active
+     * Aergo team will review the pull request and discuss on GitHub.
+ 3. Develop
+     * If idea is accepted, Aergo Team starts to develop.
```
Comment (graup) [inline AIPS/aip-1.md:23]:
```suggestion
     * If idea is accepted, Aergo team starts to develop.
```

**Review by graup — CHANGES_REQUESTED** [review]

**Pull request merged**